### PR TITLE
Restmessage monday

### LINKED ISSUE #33: Add JSDoc for RESTMessageV2.d.ts

### PR DESCRIPTION
**Pull request by emyrold**
finished, close #33

### CODE REVIEW

File: london/RESTMessageV2.d.ts [hunk 1]
```diff
@@ -1,46 +1,511 @@
-declare namespace sn_ws {
```
Comment (johncaruso) [inline london/RESTMessageV2.d.ts]: This should have been left in. See the contructor: var sm = new **sn_ws**.RESTMessageV2();

Without declaring the namespace, we don't know RESTMessageV2 is within **sn_ws**.

File: london/RESTMessageV2.d.ts [hunk 2]
```diff
@@ -1,46 +1,511 @@
...
-    setRequestBody(body: string): void;
-    setRequestBodyFromAttachment(sys_id: string): void;
-    setRequestBodyFromStream(stream: object): void;
-    setRequestHeader(name: string, value: string): void;
-    setRequestorProfile(requestorContext: string, requestorId: string): void;
-    setStringParameter(name: string, value: string): void;
-    setStringParameterNoEscape(name: string, value: string): void;
-  }
-
-  type RestHTTPMethods =
```
Comment (johncaruso) [inline london/RESTMessageV2.d.ts]: This type should not have been removed. It is referenced, e.g., in this contructor: ` constructor(name: string, methodName: RestHTTPMethods);`

**Comment by johncaruso** [comment]
Also, see the for formatting. Note the blank line after the description and before @params. Note no blank line before @example.
```
  /**
   * Gets the duration in the specified format.
   *
   * @param format The duration format.
   * @returns The current duration in the specified format.
   * @example
   *
   * var dur = new GlideDuration('3 22:00:00');
   * gs.info(dur.getByFormat('HH:mm'));
   */
  getByFormat(format: string): string;
``

File: london/RESTMessageV2.d.ts (hunk 1 above)
Comment (emyrold) [reply london/RESTMessageV2.d.ts]: maybe we can spend 2-3 mins at the office on this...

File: london/RESTMessageV2.d.ts (hunk 2 above)
Comment (emyrold) [reply london/RESTMessageV2.d.ts]: ok, I have had one too many [Koval Rye's](https://thewhiskeywash.com/whiskey-styles/american-whiskey/whiskey-review-koval-single-barrel-rye-whiskey/) tonight... maybe we can sync up in the morning...   :-)

**Pull request merged**